tests/integration/test_vulnerability_detector/test_providers/test_download_feeds.py

```diff
@@ -88,19 +86,23 @@ def test_download_feeds(get_configuration, configure_environment, restart_module
     """
     check_apply_test({'test_download_feeds'}, get_configuration['tags'])
 
-    provider = get_configuration['metadata']['provider_name']
     os = (get_configuration['metadata']['os']).capitalize()
     system_log = get_configuration['metadata']['system_log']
     download_timeout = get_configuration['metadata']['download_timeout']
+    callback_system_log = system_log
+
+    # If provider configuration has <os> tag
+    if os:
+        callback_system_log = f"{system_log} {os}"
```

Comment:
This code may trigger a KeyError exception. If there isn't any `os` tag in the configuration, line 89 will raise the exception.

```suggestion
    system_log = get_configuration['metadata']['system_log']
    download_timeout = get_configuration['metadata']['download_timeout']

    # If provider configuration has <os> tag
    try:
        callback_system_log = f"{system_log} {get_configuration['metadata']['os'].capitalize()}"
    except KeyError:
        callback_system_log = system_log
```

Reply:
It is not necessary, there will always be an `os`